lonboard/_map.py

```diff
@@ -100,7 +101,29 @@ def __init__(
         if isinstance(layers, BaseLayer):
             layers = [layers]
 
+        def _handle_anywidget_dispatch(
+            widget: ipywidgets.Widget, msg: Union[str, list, dict], buffers: List[bytes]
+        ) -> None:
+            if msg.get("kind") != "on-click":
+                return
+            self._click_handlers(tuple(msg.get("coordinate")))
```

Comment:
I'd rather flip these:
```suggestion
            if msg.get("kind") != "on-click":
                return self._click_handlers(tuple(msg.get("coordinate")))
```

That makes it easier in the future to add other kind of message callbacks

Reply:
Good call, I agree that it would be better to only send the message if there is a callback registered.  

I'm not following you on the  
> `on_click_handlers` as a trait on `BaseLayer`

why would we put it on the BaseLayer, not keep it on the map?